addons/account/models/account_move_line.py

```diff
@@ -108,6 +108,7 @@ class AccountMoveLine(models.Model):
         compute='_compute_debit_credit', inverse='_inverse_credit', store=True, precompute=True,
         currency_field='company_currency_id',
     )
+    amount = fields.Monetary(string='Amount', compute='_compute_amount', store=True, currency_field='company_currency_id')
```

Comment:
Are you certain you need to store it ?

Reply:
Positive: non-stored fields cannot used in search views.

Reply:
why not use balance field, already stored, and overwrite the search() instead ? That would avoid storing another field for a single use case... given the size of the table, I think it would be preferable.

Note that, I think the solution should change the search only for equality searches: abs(balance) == abs(value) (if people aren't aware we use abs() then putting a minus sign there will never give results, which would be unexpected (also True in your current implementation)